iron-validator-behavior.d.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   iron-validator-behavior.html
+ */
+
+/// <reference path="../polymer/types/polymer.d.ts" />
+/// <reference path="../iron-meta/iron-meta.d.ts" />
+
+declare namespace Polymer {
+
+  /**
+   * Use `Polymer.IronValidatorBehavior` to implement a custom input/form validator. Element
+   * instances implementing this behavior will be registered for use in elements that implement
+   * `Polymer.IronValidatableBehavior`.
+   */
+  interface IronValidatorBehavior {
+    ready(): void;
+
+    /**
+     * Implement custom validation logic in this function.
+     *
+     * @param values The value to validate. May be any type depending on the validation logic.
+     * @returns true if `values` is valid.
+     */
+    validate(values: object|null): Boolean|null;
```

Comment:
This should be `boolean` not `Boolean`, though the error is in the original source.

Reply:
Done.